### PR TITLE
Replace jsonwebtoken with jose

### PR DESCRIPTION
**Pull request by lukasIO**
Builds on top of and 
closes https://github.com/livekit/server-sdk-js/pull/59

### CODE REVIEW

File: src/WebhookReceiver.test.ts [hunk 1]
```diff
@@ -14,8 +14,8 @@ describe('webhook receiver', () => {
   const token = t.toJwt();
   const receiver = new WebhookReceiver(testApiKey, testSecret);
 
-  it('should receive and decode WebhookEvent', () => {
-    const event = receiver.receive(body, token);
+  it('should receive and decode WebhookEvent', async () => {
+    const event = await receiver.receive(body, await token);
```
[the remark this answers is not in the document]
Comment (lukasIO) [reply src/WebhookReceiver.test.ts:18]: yeah, `t.toJwt()` above is async

[the remark this answers is not in the document]
Comment (lukasIO) [reply src/WebhookReceiver.test.ts:18]: looks awkward, will move the await to the method call

File: src/AccessToken.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ export class AccessToken {
 
   identity?: string;
 
-  ttl?: number | string;
+  ttl: number | string;
```
[the remark this answers is not in the document]
Comment (davidzhao) [reply src/AccessToken.ts:40]: ah ha.. sorry, didn't expand to see the full context.

File: src/WebhookReceiver.test.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (davidzhao) [reply src/WebhookReceiver.test.ts]: ohhh hah, another context issue.

**Pull request merged**